tests/session/session-endSession-001.phpt

```diff
@@ -0,0 +1,75 @@
+--TEST--
+MongoDB\Driver\Session::endSession() Calling methods after session has been ended
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+$sessionA = $manager->startSession();
+$sessionA->endSession();
+
+echo throws(function() use ($sessionA) {
+    $sessionA->startTransaction();
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
+
+echo throws(function() use ($sessionA) {
+    $sessionA->abortTransaction();
+}, 'MongoDB\Driver\Exception\UnexpectedValueException'), "\n";
+
+echo throws(function() use ($sessionA) {
+    $sessionA->startTransaction();
```

Comment:
I know I'm testing ``startTransaction`` twice. But it makes the file test run "normally" if ``endSession`` wasn't called.

Reply:
Added